client/src/settings/app.tsx

```diff
@@ -247,7 +245,7 @@ function Settings({
             <a href={window.location.pathname}>Reload page to try again.</a>
           </div>
         )}
-
+        <label htmlFor="id_locale">Change language</label>
```

Comment:
You can also do this if you wanted to hide the label for everyone but screen readers.

```suggestion
        <label htmlFor="id_locale" className="visually-hidden">Change language</label>
```